### PR TITLE
improve: Set 90 day expiry on redis keys

### PR DESCRIPTION
**Pull request by nicholaspai**
RedisClient currently stores two keys:
- block for timestamp
- utilizations at block

These are used by ConfigStoreClient to compute a realized lp fee % and by dataworker to construct spoke pool clients' `fromBlocks`. They're also used by the validateRootBundle script which shouldnt ever be used to validate bundles older than 90 days

Expiring these keys after 90 days seems very safe assuming bundle block ranges stay steady

RedisJS [documentation](https://github.com/redis/node-redis#redis-commands)

Redis.API [documentation](https://redis.io/commands/set/)

### CODE REVIEW

File: src/utils/RedisUtils.ts [hunk 1]
```diff
@@ -8,6 +8,16 @@ export type RedisClient = ReturnType<typeof createClient>;
 // Current time must be >= 5 minutes past the event timestamp for it to be stable enough to cache.
 export const REDIS_CACHEABLE_AGE = 300;
 
+export async function setRedisKey(
+  key: string,
+  val: string,
+  expirySeconds: number,
```
Comment (pxrl) [inline src/utils/RedisUtils.ts:14]: Might be nice to have this default to 0, in which case the expiry is not set for this key?

File: src/utils/RedisUtils.ts [hunk 2]
```diff
@@ -11,8 +11,8 @@ export const REDIS_CACHEABLE_AGE = 300;
 export async function setRedisKey(
   key: string,
   val: string,
-  expirySeconds: number,
-  redisClient: RedisClient
+  redisClient: RedisClient,
+  expirySeconds = 0,
 ): Promise<void> {
   // EX: Expire key after expirySeconds.
   await redisClient.set(key, val, { EX: expirySeconds });
```
Comment (pxrl) [inline src/utils/RedisUtils.ts:18]: Sorry, I was unclear about the expirySeconds idea. EX = 0 is not a valid input, so I think the EX key would need to be conditional on expirySeconds being > 0.

Full disclosure on this input: If we had a generic redis wrapper/interface module then I think the ability to specify no expiry would be important, but our usage here is pretty specific...so I'm not sure whether this is ever going to be used in practice...

```
127.0.0.1:6379> SET testkey 5 EX 0
(error) ERR invalid expire time in 'set' command
```

Comment (nicholaspai) [reply src/utils/RedisUtils.ts:18]: good point

**Review by pxrl — APPROVED** [review]
No further comments :+1:

**Pull request merged**